lib/telegram/bot/updates_poller.rb

```diff
@@ -77,11 +77,26 @@ def fetch_updates
 
       def reload!
         return yield unless reload
-        ActionDispatch::Reloader.prepare!
-        if controller.is_a?(Class) && controller.name
-          @controller = Object.const_get(controller.name)
+        reloading_code do
+          if controller.is_a?(Class) && controller.name
+            @controller = Object.const_get(controller.name)
+          end
+          yield
+        end
+        Rails.application.reloader.reload!
+        yield
+      end
+
+      if defined?(Rails) && Rails.application.respond_to?(:reloader)
+        def reloading_code
+          Rails.application.reloader.reload!
```

Comment:
It's seems to be more complex than just `.reload!` https://github.com/rails/rails/blob/master/activesupport/lib/active_support/execution_wrapper.rb#L71